### DISCUSSION TITLE
HTTP Interface

### DISCUSSION THREAD

**Discussion by AktharAzif** [planetscale/discussion#99] [Feature Requests] [answered]
HTTP Interface for serverless platforms which only supports http(Cloudflare Workers).

**Comment by AktharAzif** [comment]


**Reply by fooforge** [reply]
Thanks for sharing that link, @AktharAzif! 🙂

**Comment by isaac-mcfadyen** [comment, marked as answer]
This is now a thing with the [Planetscale Serverless Driver](https://planetscale.com/blog/introducing-the-planetscale-serverless-driver-for-javascript)! 🎉
